content/en/plugins/github-api/release-notes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: GiHub API Plugin Release Notes
+linkTitle: Release Notes
+weight: 99
+description: >
+  Github API Plugin for Spinnaker release notes.
+---
+
+## Release notes
+
+### v1.0 
+
+Date: September 2023
+
+Initial release
+
+#### Known issues
+
+* The GitHub API plugin conflicts with the Armory.Header plugin's default version installed in Armory Continuous Deployment 2.30.x. 
+
+  * **Fix**: Update the Armory.Header plugin to v0.2.0+
```

Comment:
Here is the example on updating the Header plugin to 0.2.0 to be used with the Github APi plugin:
```yaml
    profiles:
      gate:
        spinnaker:
          extensibility:
            repositories:
              pluginRepository:
                url: https://raw.githubusercontent.com/armory-plugins/pluginRepository/master/repositories.json
            plugins:
              Armory.GithubApi:
                enabled: true
                version: <version>
              Armory.ArmoryHeader:
                enabled: true
                version: 0.2.0
            deck-proxy:
              enabled: true
              plugins:
                Armory.GithubApi:
                  enabled: true
                  version: <version>
                Armory.ArmoryHeader:
                  enabled: true
                  version: 0.2.0
...
```